docs/concepts/fields.md

```diff
@@ -54,30 +106,42 @@ print(user.username)
 The `data` argument will *only* contain the already validated data, based on the [order of model fields](./models.md#field-ordering)
 (the above example would fail if `username` were to be defined before `email`).
 
+### Mutable default values
+
+A common source of bugs in Python is to use a mutable object as a default value for a function or method argument,
+as the same instance ends up being reused in each call.
 
-## Using `Annotated`
```

Comment:
Replaced by the new section, which is a bit more generic in the sense that it doesn't restrict to `Field`. The below example wasn't a good one either, as `default_factory` inside annotated is not understood by type checkers.